lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_retrospectives.py

```diff
@@ -182,7 +182,11 @@ def create_encounter(db: DatabaseSession,
     if not encounter_location_references:
         return None, None
 
-    hospitalization = create_encounter_hospitalization(record)
+    try:
+        hospitalization = create_encounter_hospitalization(record)
+    except UnknownHospitalDischargeDisposition as exc:
+        LOG.warning(exc)
```

Comment:
`exc` is a bit unconventional here; typically in python we see it called `exception` or simply, `e`.

Reply:
I like "e" !